iphone/TitaniumKit/TitaniumKit/Sources/API/TiUIView.m

```diff
@@ -1609,12 +1612,7 @@ - (void)processTouchesEnded:(NSSet *)touches withEvent:(UIEvent *)event
     // Click handling is special; don't propagate if we have a delegate,
     // but DO invoke the touch delegate.
     // clicks should also be handled by any control the view is embedded in.
```

Comment:
This comment still sounds reasonable. Have you tested all other click cases as well?

Reply:
but hierarchically the gesturer event will propagate eventually.